lib/Sema/BuilderTransform.cpp

```diff
@@ -364,68 +364,70 @@ class BuilderClosureVisitor
     }
   }
 
-  VarDecl *visitBraceStmt(BraceStmt *braceStmt) {
-    SmallVector<Expr *, 4> expressions;
+  void visitBraceElement(ASTNode node, SmallVectorImpl<Expr *> &expressions) {
```

Comment:
Let’s not do this as BuilderClosureTransform and all relative logic is going away completely soon - https://github.com/apple/swift/pull/60419